packages/replayio/src/commands/record.ts

```diff
@@ -102,11 +109,18 @@ async function record(url: string = "about:blank") {
     }
 
     console.log(""); // Spacing for readability
+  } else if (unusableRecordings.length > 0) {
+    // If there were unusable recordings we should provide explicit messaging about them
+    const reason = getRecordingUnusableReason(processGroupId);
```

Comment:
q: why can't we just grab `unusableRecordings.map(r => r.unusableReason)`?

Reply:
That's fair I guess. When I first wrote this method I thought I might have to read logs or do something more involved. I'll inline it.